sippy-ng/src/releases/ReleasePayloadTable.js

```diff
@@ -124,13 +124,43 @@ function ReleasePayloadTable(props) {
     },
     {
       field: 'reject_reason',
-      headerName: 'Reject reason',
+      headerName: 'Reject reasons',
       flex: 1.5,
       hide: props.briefTable,
       renderCell: (params) => {
+        let display_reasons = []
+
+        // Until we migrate to just reject_reasons, we'll display the current value
+        // of reject_reason if present.
+        if (params.row.reject_reason) {
+          display_reasons.push(params.row.reject_reason)
+        }
+
+        // Since the reject_reason will be the first element, we only need anything after that.
+        if (params.row.reject_reasons && params.row.reject_reasons.length > 1) {
```

Comment:
the field used is `reject_reason`. Either switch the field to `reject_reasons` to accomomdate this changes

Reply:
For now, we still need `reject_reasons` for the previous payloads (the ones before my change is merged).  Although I look for `reject_reason` above, the row returned from the api contains all fields including `reject_reason` and `reject_reasons` so I'm able to consider both when determining what to display.